src/components/FlatList/index.android.tsx

```diff
@@ -0,0 +1,64 @@
+import {useFocusEffect} from '@react-navigation/native';
+import React, {ForwardedRef, forwardRef, useCallback, useState} from 'react';
+import {FlatList, FlatListProps} from 'react-native';
+
+type CustomFlatListProps = {
+    /** Same as for FlatList */
+    onScroll?: (...args: unknown[]) => unknown;
+
+    /** Same as for FlatList */
+    onLayout?: () => (...args: unknown[]) => unknown;
```

Comment:
I think these parameters are already included in `FlatListProps` type.

Reply:
Yes, we can remove `onScroll` and `onLayout` (and `innerRef` like @VickyStash suggested in the other comment).